tools/packaging/release/release.sh

```diff
@@ -51,56 +49,9 @@ function _check_required_env_var()
 	return 0
 }
 
-function _next_release_version()
+function _release_version()
 {
-	_check_required_env_var "RELEASE_TYPE"
-
-	local current_release=$(cat "${repo_root_dir}/VERSION")
-	local current_major
-	local current_everything_else
-	local next_major
-	local next_minor
-
-	IFS="." read current_major current_minor current_everything_else <<< ${current_release}
-
-	case ${RELEASE_TYPE} in
-		major)
-			next_major=$(expr $current_major + 1)
-			next_minor=0
-			;;
-		minor)
-			next_major=${current_major}
-			# TODO: As we're moving from an alpha release to the
-			# new scheme, this check is needed for the very first
-			# release, after that it can be dropped and only the
-			# else part can be kept.
-			if grep -qE "alpha|rc" <<< ${current_everything_else}; then
-				next_minor=${current_minor}
-			else
-				next_minor=$(expr $current_minor + 1)
-			fi
-			;;
-		*)
-			_die "${RELEASE_TYPE} is not a valid release type, it must be: major or minor"
-			;;
-	esac
-
-	next_release_number="${next_major}.${next_minor}.0"
-	echo "${next_release_number}"
-}
-
-function _update_version_file()
-{
-	_check_required_env_var "RELEASE_VERSION"
-
-	git config user.email "katacontainersbot@gmail.com"
-	git config user.name "Kata Containers Bot"
-
-	echo "${RELEASE_VERSION}" > "${repo_root_dir}/VERSION"
-	git diff
-	git add "${repo_root_dir}/VERSION"
-	git commit -s -m "release: Kata Containers ${RELEASE_VERSION}"
-	git push
+	cat "${repo_root_dir}/VERSION"
```

Comment:
FYI - the failure in my command is that it seems to ignore pre-releases, so at the moment we'd successful pass when it should fail. I'm not sure with the lack of alpha is we are likely to use pre-releases much going forwards though?

Reply:
The release would fail straight away if we're trying to push a tag that already exists.  With this in mind, I sincerely think we can just ignore the check.

About pre-releases, I don't think we'll use them at all from now on.